recipe/meta.yaml

```diff
@@ -21,17 +21,20 @@ requirements:
     - python
     - pip
     - cython >=0.28.5
-    - numpy
+    - numpy >=1.13.3
```

Comment:
Because of the nature of the build, we will probably want to pin to a specific version (probably the pin from aggregate (cbc)).

Reply:
ok got it. So remove pinning?